contrib/mkimage-archarm-pacman.conf

```diff
@@ -0,0 +1,98 @@
+#
+# /etc/pacman.conf
+#
+# See the pacman.conf(5) manpage for option and repository directives
+
+#
+# GENERAL OPTIONS
+#
+[options]
+# The following paths are commented out with their default values listed.
+# If you wish to use different paths, uncomment and update the paths.
+#RootDir     = /
+#DBPath      = /var/lib/pacman/
+#CacheDir    = /var/cache/pacman/pkg/
+#LogFile     = /var/log/pacman.log
+#GPGDir      = /etc/pacman.d/gnupg/
+HoldPkg     = pacman glibc
+#XferCommand = /usr/bin/curl -C - -f %u > %o
+#XferCommand = /usr/bin/wget --passive-ftp -c -O %o %u
+#CleanMethod = KeepInstalled
+#UseDelta    = 0.7
+Architecture = armv7h
```

Comment:
@tianon Yes, it will work only in/for ```armv7```. Arch Linux ARM supports also other ARM architectures (v5, v6 and v8) but as noted in https://github.com/docker/docker/pull/16704#issuecomment-163181320, other arches support only very old SBCs (which i'm not sure if docker would be able to run on).

Adding support for other arches would be very easy and can be done in two ways:

* Adding a ```pacman.conf``` for every arch we would like to support
* Modifing the ```mkimage-arch.sh``` script to detect arch and generate the right ```pacman.conf``` file at runtime.

Let me know if you would like to add support for other arches and which method you would like the best.